ci/runtests.sh

```
@@ -0,0 +1,44 @@
+
+function setup_env() {
+  cd /app
+  # this is a throwaway container, venv not needed!
+
+  ls -la
+  cat requirements.txt requirements-dev.txt
+  pip install -r requirements-dev.txt
+}
+
+function main() {
+  local onlysetup=$1
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)